GAE/src/org/waterforpeople/mapping/dataexport/GraphicalSurveySummaryExporter.java

```diff
@@ -616,10 +620,11 @@ private void writeAnswer(
             Row row,
             int startColumn,
             QuestionDto questionDto,
-            final String value,
+            String value,
             boolean useQuestionId) {
 
         assert value != null;
+        value = value.trim();
```

Comment:
Just a matter of preference here but perhaps its better to have the `trim()` right at the first point of data entry into the system.. in this case the `BulkDataServiceClient`?